[CLI] 4.1 `module not found`

<!--

Please note this is an issue tracker, not a support forum.
For general questions, please use StackOverflow or Slack.

For bugs, please fill out the template below.

-->

## What you are doing?
I have a "pretest" script that resets the DB from seeds.
`"pretest": "sequelize db:migrate:undo:all && sequelize db:migrate && sequelize db:seed:all && npm run lint"`

After `sequelize-cli` 4.1 this no longer works for me. I have to use local install of sequelize-cli.
`"pretest": "node_modules/.bin/sequelize db:migrate:undo:all && node_modules/.bin/sequelize db:migrate && node_modules/.bin/sequelize db:seed:all && npm run lint"`

Running `sequelize` (to get help menu) doesn't work either.

<!-- Post a minimal, self-contained code sample that reproduces the issue, include config and migration file(s) -->

## What do you expect to happen?
I expect globally installed sequelize-cli to work.

## What is actually happening?
```
▶ sequelize
module.js:545
    throw err;
    ^

Error: Cannot find module 'sequelize'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:470:25)
    at Module.require (module.js:593:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/sequelize-cli/lib/helpers/model-helper.js:7:18)
    at Module._compile (module.js:649:30)
    at Object.Module._extensions..js (module.js:660:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:501:12)
    at Function.Module._load (module.js:493:3)
    at Module.require (module.js:593:17)
    at require (internal/module.js:11:18)
    at /usr/local/lib/node_modules/sequelize-cli/lib/helpers/index.js:18:52
    at Array.forEach (<anonymous>)
    at Object.<anonymous> (/usr/local/lib/node_modules/sequelize-cli/lib/helpers/index.js:17:4)
    at Module._compile (module.js:649:30)
    at Object.Module._extensions..js (module.js:660:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:501:12)
    at Function.Module._load (module.js:493:3)
    at Module.require (module.js:593:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/sequelize-cli/lib/commands/init.js:7:16)
    at Module._compile (module.js:649:30)
    at Object.Module._extensions..js (module.js:660:10)
    at Module.load (module.js:561:32)
    at tryModuleLoad (module.js:501:12)
    at Function.Module._load (module.js:493:3)
```

_Output, either JSON or SQL_


__Dialect:__ mysql
__Database version: mysql  Ver 14.14 Distrib 5.7.21, for osx10.13 (x86_64) using  EditLine wrapper
__Sequelize CLI version: 4.1.0
__Sequelize version: 4.38.0

--- a/src/helpers/model-helper.js
+++ b/src/helpers/model-helper.js
@@ -1,6 +1,6 @@
 import helpers from './index';
-import { DataTypes } from 'sequelize';
 
+const Sequelize = helpers.generic.getSequelize();
 const validAttributeFunctionType = 'array';
 
 /**
@@ -8,7 +8,7 @@ const validAttributeFunctionType = 'array';
  * @param {string} dataType
  */
 function validateDataType (dataType) {
-  if (!DataTypes[dataType.toUpperCase()]) {
+  if (!Sequelize.DataTypes[dataType.toUpperCase()]) {
     throw new Error(`Unknown type '${dataType}'`);
   }
 

